IdentityCore/src/broker_operation/request/token_request/MSIDBrokerOperationTokenRequest.m

```diff
@@ -146,7 +147,7 @@ - (NSDictionary *)jsonDictionary
     json[MSID_BROKER_ACCOUNT_HOME_TENANT_ID] = self.accountHomeTenantId;
     json[MSID_CLIENT_SKU_KEY] = self.clientSku;
     json[MSID_SKIP_VALIDATE_RESULT_ACCOUNT_KEY] = [@(self.skipValidateResultAccount) stringValue];
-    
+    json[MSID_FORCE_REFRESH_KEY] = [@(self.forceRefresh) stringValue];
```

Comment:
Can you please also verify when MSAL and broke version mismatches, the new passed in params will not cause issues?

Reply:
I think there should be no problem because the object will be deserialized (and vice-versa) to a json string, so missing any optional properties isn't a trouble, the receiving lib just don't know the new property exist. I'm not sure how to verify it though...